suplemon/logger.py

```diff
@@ -33,6 +33,8 @@ def close(self):
         self.acquire()
         try:
             for record in self.buffer:
+                if record.levelno < self.level:
```

Comment:
I'm not sure what you mean by the default behaviour. I needed to add this to filter records at print time. Without this some records slipped through to the console because `debug_level` hadn't been loaded from config yet. It just skips printing any record with a lower level than the handlers level.